### PR TITLE
Always use printf with %b\n unless a different var is needed

### PR DESCRIPTION
**Pull request by nnyyxxxx**
## Type of Change
- [x] Refactoring

## Description
Always use "%b\n" unless a different var is needed and always put the new lines before the actual message inside of "%b\n" as recommended by shellcheck

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Review by adamperkowski — APPROVED** [review]

**Comment by nnyyxxxx** [comment]
Ill fix these conflicts

**Comment by ChrisTitusTech** [comment]
Update with new merges as we changed file names

**Comment by nnyyxxxx** [comment]
> Update with new merges as we changed file names

finished

**Comment by nnyyxxxx** [comment]
all conflicts fixed again

File: core/tabs/utils/ssh.sh [hunk 1]
```diff
@@ -23,12 +23,11 @@ ask_for_host_details() {
...
-    printf "%b\n" "    HostName $host" >> ~/.ssh/config
-    printf "%b\n" "    User $user" >> ~/.ssh/config
-    printf "%b\n" "    IdentityFile ~/.ssh/id_rsa" >> ~/.ssh/config
-    printf "%b\n" "    StrictHostKeyChecking no" >> ~/.ssh/config
-    printf "%b\n" "    UserKnownHostsFile=/dev/null" >> ~/.ssh/config
+    {
+        printf "%b\n" "Host $host_alias"
+        printf "%b\n" "    HostName $host"
+        printf "%b\n" "    User $user"
+    } >> ~/.ssh/config
```
Comment (ChrisTitusTech) [inline core/tabs/utils/ssh.sh:30]: Need Identify File and other options.

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**